modules/aws/centos-gfx/centos-gfx-startup.sh.tmpl

```diff
@@ -153,24 +152,24 @@ enable_persistence_mode() {
 install_pcoip_agent() {
     if ! (rpm -q pcoip-agent-graphics)
     then
-        log "--> Start to install PCoIP agent ..."
-        # Get the Teradici pubkey
-        log "--> Get Teradici pubkey"
-        rpm --import https://downloads.teradici.com/rhel/teradici.pub.gpg
+        log "--> Starting PCoIP agent installation..."
 
-        # Get PCoIP repo
-        log "--> Get Teradici PCoIP agent repo"
-        wget --retry-connrefused --tries=3 --waitretry=5 -O /etc/yum.repos.d/pcoip.repo https://downloads.teradici.com/rhel/pcoip.repo
+        log "--> Getting Teradici pubkey..."
+        teradici_pubkey=teradici.pub.gpg
```

Comment:
this variable doesn't seem to accomplish anything since it's not used anywhere else.